lib/autoyast.pm

```diff
@@ -702,7 +702,7 @@ sub expand_variables {
     # Expand other variables
     my @vars = qw(SCC_REGCODE SCC_REGCODE_HA SCC_REGCODE_GEO SCC_REGCODE_HPC
       SCC_REGCODE_LTSS SCC_REGCODE_WE SCC_URL ARCH LOADER_TYPE NTP_SERVER_ADDRESS
-      REPO_SLE_MODULE_DEVELOPMENT_TOOLS);
+      REPO_SLE_MODULE_DEVELOPMENT_TOOLS SCC_REGCODE_LIVE);
```

Comment:
we use `get_var`, so we might not need to expand the variable, right?

Reply:
It didn't work without this change for SCC_REGCODE_LIVE